optuna/study/study.py

```diff
@@ -410,13 +399,11 @@ def objective(trial):
                 Currently, progress bar is experimental feature and disabled
                 when ``n_trials`` is :obj:`None`, ``timeout`` not is :obj:`None`, and
                 ``n_jobs`` :math:`\\ne 1`.
-
-        Raises:
-            RuntimeError:
-                If nested invocation of this method occurs.
```

Comment:
<img width="926" alt="Screenshot 2022-10-11 17 36 23" src="https://user-images.githubusercontent.com/5564044/195040959-a7628191-a286-4846-91c1-50b09b5bc2f6.png">

I tried to update the docstring like above to let users know nested invocation of `study.optimize()` is prohibited. But on second thought, I'm not sure if it is worth noting in the documentation.

@hvy Do you have any ideas or can we keep the documentation as it is?

Reply:
Good point. I actually think we can omit it from the documentation as well, as I can't see valid use cases for trying to do so, and the behavior after this change is rather expected. @sile might know better about why we should, or should not, document it, though.

Reply:
> I actually think we can omit it from the documentation as well

Agreed😆